test/sym/div_f32.wat

```diff
@@ -0,0 +1,22 @@
+(module
+  (import "symbolic" "f32_symbol" (func $f32_symbol (result f32)))
+  (import "symbolic" "assume" (func $assume_i32 (param i32)))
+  (import "symbolic" "assert" (func $assert_i32 (param i32)))
+
+  (func $start
+    (local $x f32)
+    (local $y f32)
+
+    (local.set $x (call $f32_symbol))
+    (local.set $y (call $f32_symbol))
+
+    ;; x = y
+    (call $assume_i32 (f32.eq (local.get $x) (local.get $y)))
+    ;; 1/x = 1/y
+    (call $assert_i32 (f32.eq
+      (f32.div (f32.const 1) (local.get $x))
+      (f32.div (f32.const 1) (local.get $y))))
+  )
```

Comment:
I think a better test would be:

```txt
(x / y) = (x / y)
```

Which is false when `x / y` is NaN, because NaN is not equal to NaN...

Reply:
I declined this test with `add_f...` `sub_f...` `mul_f...` ...